### ISSUE TITLE
Simple UI for old people

### ISSUE THREAD

**Issue by aguaviva** [TeamNewPipe/NewPipe#7606]
Hide the config settings and anything that our mother wouldn't understand, need or could confuse her. 

I could list a few examples but I think we all know what those bits are :)

This will make our elder people lives easier, and thanks for this great app!

**Comment by yashpalgoyal1304** [comment]
yeah, i am also in favour of making it friendly for older people. but the speed mentioning title got took away in the v0.21.14 :(

https://github.com/TeamNewPipe/NewPipe/pull/7363#issuecomment-962605601

**Comment by aguaviva** [comment]
@yashpalgoyal1304 all those bits should be configurable if possible.

@TiA4f8R Sorry for ignoring the template, if you think it is crucial I will be happy to fill it.

**Comment by TiA4f8R** [comment]
Yes, please fill in the template.

**Comment by aguaviva** [comment]
### Checklist


- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### What feature do you want?

NewPipe's UI is very rich and powerful, while most of us love it many old people might find it challenging. I'd be great to be able to disable certain UI elements so old people can still use and enjoy this fantastic app.

#### Why do you want this feature?

I guess we all have old people in our families that use tablets and phones, and as they get older they find harder to use them. I'd be great to give them this treat so they can still watch videos of the things they love.
